template/en/default/bug/new_bug.html.tmpl

```diff
@@ -5,15 +5,27 @@
 # This Source Code Form is "Incompatible With Secondary Licenses", as
 # defined by the Mozilla Public License, v. 2.0.%]
 
-[% PROCESS "global/field-descs.none.tmpl" %]
-
 [% title = BLOCK %]Enter [% terms.Bug %] [% END %]
 
-[% PROCESS global/header.html.tmpl
-  title = title
-  generate_api_token = 1
-  no_yui = 1
-%]
+[% PROCESS global/variables.none.tmpl %]
+[% IF !header_done %]
+  [% PROCESS bug_modal/common-header.html.tmpl %]
```

Comment:
Alternatively, we could use _'s. I think _ is slightly more common for us than -, and I guess focusing on a single style would be nice. :-)